scene/gui/graph_edit.cpp

```diff
@@ -29,6 +29,7 @@
 /**************************************************************************/
 
 #include "graph_edit.h"
+
```

Comment:
This is intetional. When I remove this line, clang-format sorts it and it ends up before `#include "graph_edit.h"` which then results in a compilation error.

Reply:
That's very odd! It works everywhere else

Reply:
I just checked it with the other files, same for me there (using clang-format 17.0.6)
But I'm actually confused why it worked everywhere else in the first place since
```
#include "graph_edit.compat.inc"
#include "graph_edit.h"
```
is the lexicographically correct order.

Reply:
Unsure, some quirk possibly, but same with 17.0.6 and it works fine with this file specifically

Reply:
Mmh, I still have no idea why this doesn't work for me, but I changed it manually for now.

Reply:
Tested with 16.0.4 as well and it works as above, so probably some strange bug there, worth a look later